### ISSUE TITLE
Is it normal that the downloads are slow

### ISSUE THREAD

**Issue by Falda166** [JurajNyiri/pytapo#75]
I Download All the New recorder that the camera makes directly but the download are very slow.

What are the possible reasons?

(I have a tapo c200, its on an raspberry 3) for more data just ask

**Comment by JurajNyiri** [comment]
Yes, it downloads the recordings by playing them and it is already doing it as fast as possible.
